sriov/tests/config_template.yaml

```diff
@@ -11,3 +11,10 @@ container_volumes: "-v /sys:/sys -v /dev:/dev -v /lib/modules:/lib/modules"
 vlan: 10
 mtu: 1500
 bonding_switch_delay: 5
+testpmd_img: "testpmd:22.11.2"
+testpmd_port: 9000
+trafficgen_img: "trafficgen:latest"
+trafficgen_port: 8080
+trafficgen_ip: 10.88.0.88
```

Comment:
Normally we don't need to specify this - the trafficgen container will use its host ip address, the rely on the port mapping ( '-p' that is being used ). If the client is on the same host as the trafficgen container, then client simply points to the localhost as the server ip and the '-p' host port. If the client is from another machine, then the client will point to the trafficgen host ip as the server ip and the   '-p' host port.

Reply:
Updated in latest